hmda-data-publisher/src/main/resources/unset_cron_vars.txt

```diff
@@ -0,0 +1 @@
+unset PANEL_PRIVATE_SCHEDULE_2018 LAR_PRIVATE_SCHEDULE_2018 TS_PRIVATE_SCHEDULE_2018 LAR_PUBLIC_SCHEDULE_2018 TS_PUBLIC_SCHEDULE_2018 LAR_PUBLIC_SCHEDULE_2019 TS_PUBLIC_SCHEDULE_2019 PANEL_PRIVATE_SCHEDULE_2019 LAR_PRIVATE_SCHEDULE_2019 LAR_PRIVATE_SCHEDULE_LOAN_LIMIT_2019 TS_PRIVATE_SCHEDULE_2019 LAR_Scheduler_Quarterly_2020 TS_Scheduler_Quarterly_2020
```

Comment:
Is there a reason for this file? If this is to help with manually unsetting all variables, it should be part of README and not `resources`

Reply:
Additionally, please include a README.md for this service @PatrickGoRaft that provides some information